scripts/release/Sources/set-marketing-version/main.swift

```diff
@@ -54,8 +54,8 @@ struct MarketingVersion: ParsableCommand {
 
     func runSPMVersionUpdate() throws {
         let spmManifestURL = projectPathURL.appendingPathComponent("Tests/Integration/SPM/project.yml")
-        try replaceLineContaining("branch: release/v",
-                                  with: "    branch: release/v\(marketingVersion)",
+        try replaceLineContaining("branch: v",
```

Comment:
We're dropping `release/` prefix here, is that intentional?

Reply:
Actually, our releases are still in release/v branches, see https://github.com/mapbox/mapbox-common-ios/pull/334

Reply:
This construction will use tags, because branches may be deleted